keras_retinanet/utils/eval.py

```diff
@@ -21,9 +21,45 @@
 
 import numpy as np
 import os
-
+import io
+import tensorflow as tf
```

Comment:
`eval.py` shouldn't depend on tf, only in the paths where tf is actually required

Reply:
As TensorboardImage call tensorflow function can I create a specific file for this function ?